src/lib/commandBarConfigs/validators.ts

```diff
@@ -153,3 +153,57 @@ export const loftValidator = async ({
     return 'Unable to loft with selected sketches'
   }
 }
+
+export const shellValidator = async ({
+  data,
+}: {
+  data: { selection: Selections }
```

Comment:
Don't want to pass in selections directly?

Reply:
There https://github.com/KittyCAD/modeling-app/blob/b5ddc43bbb83020432a87d5ef28baa82ff500a41/src/machines/commandBarMachine.ts#L285